Kickstarter-iOS/Features/RewardAddOnSelection/Controller/RewardAddOnSelectionViewController.swift

```diff
@@ -176,11 +176,14 @@ final class RewardAddOnSelectionViewController: UIViewController {
 
     self.viewModel.outputs.goToPledge
       .observeForControllerAction()
-      .observeValues { data in
-        let vc = PledgeViewController.instantiate()
-        vc.delegate = self.pledgeViewDelegate
-        vc.configure(with: data)
-        self.navigationController?.pushViewController(vc, animated: true)
+      .observeValues { [weak self] data in
+        guard let self else { return }
+
+        if featurePostCampaignPledgeEnabled(), data.project.isInPostCampaignPledgingPhase {
```

Comment:
Nit: If this double-check (for the feature and the project flag) becomes used more frequently in the app, it might be nice to have a helper method like `usePostCampaignPledgeFlow(forProject: project)`